code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -183,8 +183,8 @@
 	damage_mult = CONFIG_GET(number/combat_define/base_hit_damage_mult)
 
 /obj/item/weapon/gun/pistol/b92fs/M9
-	name = "\improper M9 Custom pistol"
-	desc = "A 20th century military firearm customized for special forces use, fires tranq darts to take down enemies nonlethally"
+	name = "\improper LowSecurity1 pistol"
+	desc = "(M9) A LowSec1 tranquilizer pistol. Fires tranq darts. Manufactured by an unknown manufacturer on Mars in the 2100s, now known as Cydonia Authority, as a nonlethal alternative to Armsdealer's Concord MaxSec series. This weapon was primarily used by clandestine operatives, before it lost popularity. The rioters on Mars were able to get their hands on these in the 2100's, in albeit few numbers, when they raided an important industrial facility that still had the LowSec1 in stock. It used to be favored by the Mars rioters for its nonlethal takedown capability. Cydonia Authority is the only manufacturer that currently still produces this weapon, but only in small numbers."
```

Comment:
This one is just too long. Delete the most minuscule details before exporting to the codex.